src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -1315,6 +1319,9 @@ Status ColorControlServer::moveToHueAndSaturation(EndpointId endpoint, uint16_t
     uint16_t halfWay    = isEnhanced ? HALF_MAX_UINT16T : HALF_MAX_UINT8T;
     bool moveUp;
 
+    VerifyOrReturnValue(saturation <= MAX_SATURATION_VALUE, Status::ConstraintError);
```

Comment:
Shouldn't this check the hue as well?  Per spec it should be `max 254` in the non-enhanced case.  Followup fine for this.

`moveToHueCommand` does check the hue value...

Reply:
You are right, the non enhanced needs a check. I'll add it now.